CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+v1.9.0 (upcoming)
+-------------------
+
+- `dub run` will now automatically fetch a package if it's not found locally [pull #1428][pull1428]
```

Comment:
We could also opt to switch to the "D" `changelog` format and add individual entries.
The advantage: it would be displayed at https://dlang.org/changelog/pending.html and automatically part of the release changelog
The disadvantage: more overhead for the tiny traffic that dub currently receives